sql/2021/structured-data/summary_count.sql

```diff
@@ -0,0 +1,51 @@
+#standardSQL
+  # A summary count of the pages run against
+SELECT
+  SUM(CASE
+      WHEN `success` IS NOT NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `success`,
+  SUM(CASE
+      WHEN `errors` IS NOT NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `errors`,
+  SUM(CASE
+      WHEN `success` IS NOT NULL AND `errors` IS NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `success_no_errors`,
+  SUM(CASE
+      WHEN `errors` IS NOT NULL AND `success` IS NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `errors_no_success`,
+  SUM(CASE
+      WHEN `success` IS NOT NULL AND `errors` IS NOT NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `success_errors`,
+  SUM(CASE
+      WHEN `success` IS NULL AND `errors` IS NULL THEN 1
+    ELSE
+    0
+  END
+    ) AS `no_success_no_errors`
+FROM (
+  SELECT
+    JSON_EXTRACT(`structured_data`,
+      '$.structured_data') AS `success`,
+    JSON_EXTRACT(`structured_data`,
+      '$.log') AS `errors`
```

Comment:
I'd suggest evaluating these to booleans, which you can negate as needed in the outer SELECT, to avoid the need to repeatedly do the IS [NOT] NULL checks.

Also, are the backticks necessary?
```suggestion
    JSON_EXTRACT(`structured_data`, '$.structured_data') IS NOT NULL AS is_success,
    JSON_EXTRACT(`structured_data`, '$.log') IS NOT NULL AS has_errors
```